snapcraft/internal/pluginhandler/_plugin_loader.py

```diff
@@ -29,10 +30,19 @@
 
 
 def load_plugin(
-    plugin_name, part_name, project_options, properties, part_schema, definitions_schema
+    plugin_name,
+    part_name,
+    project_options,
+    properties,
+    part_schema,
+    definitions_schema,
+    local_plugins_dir=None,
```

Comment:
mind adding type annotation for this new entry? Optional[str] I believe it is...

This final comma here might break flake8, let's see from the tests.

Reply:
Added type annotation.